### ISSUE TITLE
Source maps reference non-existent source files

### ISSUE THREAD

**Issue by skbergam** [samchon/typia#387]
## Bug Report
When referencing this package from a `react-scripts` project, we see several warnings because of invalid source maps:
```
WARNING in ../../node_modules/typescript-json/lib/schemas/IJsonSchema.js
Module Warning (from ../../node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/sbergam/git/incident-management-service/node_modules/typescript-json/src/schemas/IJsonSchema.ts' file: Error: ENOENT: no such file or directory, open '/Users/sbergam/git/incident-management-service/node_modules/typescript-json/src/schemas/IJsonSchema.ts'
```

This discussion describes the issue (and how to fix it):
https://github.com/facebook/create-react-app/discussions/11767

**TL;DR** if your npm package contains sources maps (which typescript-json does) then you need to make sure the source files referenced in the source maps _actually_ get published with the package.

E.g. the file `node_modules/typescript-json/lib/index.js.map` as published on npm has these contents:
```
{"version":3,"file":"index.js","sourceRoot":"","sources":["../src/index.ts"],"names":[],"mappings":";;;;;;;;;;;;;;;;;;;;;;;;;;;;AAAA,6CAAiC;AAEjC,kBAAe,IAAI,CAAC;AACpB,2CAAyB"}
```
...but the file `src/index.ts` that it references is _not_ published on npm.

I hope this helps. I looked for a publish command in the repo to see if the src directory was being excluded onpurpose, but I'm not sure where it is (otherwise I would have created a PR for this issue).

**Comment by samchon** [comment]
Can you give me your opinion about two options?

1. Generate `d.ts.map` files and do not publish `js.map` files 
2. Publish `src` directory

**Comment by skbergam** [comment]
I think #2 is best.
